osmaxx/conversion/management/commands/result_harvester.py

```diff
@@ -12,7 +12,7 @@
 
 from osmaxx.conversion import models as conversion_models
 from osmaxx.conversion._settings import CONVERSION_SETTINGS
-from osmaxx.conversion_api.statuses import FINAL_STATUSES, FINISHED, FAILED
+from osmaxx.conversion.constants.statuses import FINAL_STATUSES, FINISHED, FAILED
```

Comment:
Analogous to the above comment:
`from osmaxx.conversion.constants import statuses` might be worth considering as an import and usage would become `statuses.FINAL_STATUSES`.